client/lib/plans/constants.js

```diff
@@ -466,11 +478,13 @@ export const PLANS_LIST = {
 			FEATURE_PREMIUM_SUPPORT,
 		],
 		getSignupFeatures: () => [
+			FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
 			FEATURE_PREMIUM_SUPPORT,
-			FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED_SIGNUP,
+			FEATURE_SPAM_AKISMET_PLUS,
 			FEATURE_ALL_FREE_FEATURES,
 		],
 		getBillingTimeFrame: () => i18n.translate( 'per month, billed monthly' ),
+		getSignupBillingTimeFrame: () => 'per month', //PLANS A/B TEST: Translate if test passes
```

Comment:
It is my understanding that we pay for these translations. I don't want to waste the money if the test doesn't pass. I'm happy to go wrap the translate function around strings that have been translated already but would prefer not sending the new ones over just yet.